storage_service/templates/snippets/packages_table.html

```diff
@@ -76,8 +76,19 @@
           {% if package.package_type in 'AIP AIC' and not package.replicated_package %}
             <a href="{% url 'aip_reingest' package.uuid %}?next={{ request.path }}">{% trans "Re-ingest" %}</a>
           {% endif %}
+          {% if package.package_type in package.PACKAGE_TYPE_CAN_DELETE %}
+          <a href="javascript:void(0)" class="request-delete"
```

Comment:
I vaguely remember that `javascript:void` shouldn't be used.
I've found this: https://stackoverflow.com/a/1293130.

I remember in AngularJS the preferred method was to use `href="#"` + `return false` or buttons.

Reply:
Ok, changed to use `href="#"` and `return false`.